tests/test_PostEndpoint.py

```diff
@@ -0,0 +1,26 @@
+
+
+
+from c20_server.PostEndpoint import app
+
+
+def empty_json():
+    app.config['TESTING'] = True
+    client = app.test_client()
+
+    result = client.get('/empty_json')
+    assert result.status_code == 200
+
+def get_job_success():
+    app.config['TESTING'] = True
+    client = app.test_client()
+
+    result = client.get('/get_job')
+    assert result.status_code == 200
+
+def get_job_unsuccess():
+        app.config['TESTING'] = True
+        client = app.test_client()
+
+        result = client.get('/get_job')
+        assert result.status_code == 404
```

Comment:
This test appears to be the same code as the previous test other than testing for 200 vs. 404.  How can both pass?  Am I missing something?

Reply:
I will work to fix all that.